src/platforms/macos/macosutils.mm

```diff
@@ -89,3 +89,44 @@ bool dockClickHandler(id self, SEL cmd, ...) {
 void MacOSUtils::showDockIcon() {
   [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
 }
+
+void MacOSUtils::installerStrings() {
+  //% "Mozilla VPN for macOS"
+  qtTrId("macosinstaller.title");
+
+  //% "Unable to install"
+  qtTrId("macosinstaller.unsupported_version.title");
+
+  //% "Mozilla VPN requires Mac OS X 10.6 or later."
+  qtTrId("macosinstaller.unsupported_version.message");
+
+  //% "Previous Installation Detected"
+  qtTrId("macosinstaller.previous_build.title");
+
+  //% "A previous installation of Mozilla VPN exists at /Applications/Mozilla "
```

Comment:
Please fix this, now you have a space in the middle of the file path.

Reply:
No this is right. The path is `/Applications/Mozilla VPN.app`

Reply:
Uh, completely missed that 🤦🏼